src/__tests__/server-smoke.test.ts

```diff
@@ -0,0 +1,29 @@
+import { startServer } from '../server';
+import type { Server } from 'http';
+import request from 'supertest';
+
+jest.mock("../server", () => {
+  const original = jest.requireActual("../server");
+  return {
+      ...original,
+      ebbs: jest.fn()
+  };
+});
```

Comment:
I assume this doesn't work because the `server.ts` code has an internal reference to `ebbs`, which doesn't get replaced by this mock. I'd recommend refactoring `startServer` to be more test-friendly by adding an optional argument which is a function to call to get the list of attached EBBs, defaulting to the internal `ebbs` function when not specified.

Reply:
More or less - the solution I came up with was exporting a function and then having the module import itself, which allows `jest` to mock that exported function.  

After various iterations, I think 3388136fbf1f7bd825d5e6d1435442745d62155f mocks the least amount of code / runs the most "real" code.  What do you think?